docs/qx_example.rst

```diff
@@ -1,7 +1,7 @@
 QX Simulation
 =============
 
-This tutorial explains how to compile an OpenQL program and execute it on QX. We will use the example of rolling an 8-faced dice. Rolling this dice results in 1 out of 8 outcomes. The complete code for this example is available in examples/dice.py
+This tutorial explains how to compile an OpenQL program and execute it on QX. We will use the example of rolling an 8-faced dice. Rolling this dice results in 1 out of 8 outcomes. The complete code for this example is available in ``examples/dice.py``, but you can also copy the snippits over to your own script as we walk through it.
```

Comment:
Last sentence after 'but' can be left out. IMO it does not add any new information.